great_expectations/data_context/types/resource_identifiers.py

```diff
@@ -202,12 +202,16 @@ def from_object(cls, validation_result):
 
 class GeCloudIdentifier(DataContextKey):
     def __init__(
-        self, resource_type: "GeCloudRESTResource", ge_cloud_id: Optional[str] = None
+        self,
+        resource_type: "GeCloudRESTResource",
+        ge_cloud_id: Optional[str] = None,
+        resource_name: Optional[str] = None,
     ) -> None:
         super().__init__()
 
         self._resource_type = resource_type
         self._ge_cloud_id = ge_cloud_id if ge_cloud_id is not None else ""
+        self._resource_name = resource_name if resource_name is not None else ""
```

Comment:
```suggestion
        self._resource_name = resource_name or ""
```
You can shorten these expressions to just `x or y`

Reply:
I hesitate to use this pattern since often I want to explicitly compare to None but in this case it works. Thanks! 🙇

Reply:
I have noticed we use lots of explicit `is None` / `is not None` in our codebase. 
Is there a particular reason for that? 
As general rule I tend to use the `if SOMETHING:` / `if not SOMETHING:` as it delegates the tuthy check to `__bool__`. Unless there is a specific reason to treat `None` differently.